<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCustomPlot" name="plot1" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="plot2" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCustomPlot" name="map" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>288</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>288</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="top" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>264</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Cutermur selectat:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,2,3,3,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="highDate">
          <property name="text">
           <string>data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="highMag">
          <property name="text">
           <string>magnitudine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="highLat">
          <property name="text">
           <string>latitudine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="highLong">
          <property name="text">
           <string>longitudine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="highDepth">
          <property name="text">
           <string>adancime</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
